app/Actions/Album/Setter.php

```diff
@@ -35,6 +35,8 @@ protected function __construct(Builder $query, string $property)
 	/**
 	 * @param string $albumID the ID of the album
 	 * @param mixed  $value   the value to be set
+	 *
+	 * @throws ModelNotFoundException
 	 */
 	public function do(string $albumID, mixed $value): void
```

Comment:
Can't we have this method use a Album or BaseAlbum instead ?

Reply:
I will need to reread he code globally outside the PR.

Reply:
See above. This is dead code and I did not nuke it, because I didn't know if you wanted to keep it.

Reply:
Will need to dig into it, with how to interface with Livewire.

Reply:
I would like to nuke all these superfluous, overly complicated actions. Just call the model function directly. See my [comment in the previous thread](https://github.com/LycheeOrg/Lychee/pull/1217#discussion_r825428655).

I simply haven't nuked them in this PR, because I did not want to break the Livewire part. Of course, there are a handful of actions (like archiving, rotating images) which do something relevant. These actions have been adopted to use the model objects directly instead of IDs. But 95% of the actions are a ridiculous, complicated way to express a one-liner. As the controller methods don't use these actions anymore, I haven't adopted them.